styles/_forms.scss

```diff
@@ -374,48 +375,17 @@
 }
 
 .gc-file-input-upload-button {
-  @apply bg-gray-200 border-1 border-gray-400 text-black-default font-bold py-2 px-6 rounded-lg mb-14 text-center inline-block;
-  @apply md:w-full md:text-small_base cursor-pointer border-b-4 py-3 px-5 mr-2 m-0 text-base no-underline outline-none relative;
-
-  &:focus-within {
-    box-shadow: 0 0 0 2px white;
-    outline: solid 2px #303fc3;
-    outline-offset: 2px;
-    @apply bg-blue-focus;
-    @apply text-gray-50;
-  }
-
-  &:active {
-    box-shadow: 0 0 0 2px white;
-    outline: solid 2px #1b2736;
-    outline-offset: 2px;
-  }
-  &:hover {
-    @apply bg-gray-300;
-    @apply text-black-default;
-  }
-}
-
-/**
- * Don't want to use the 'standard' method
```

Comment:
@bryan-robitaille I did try to just `display: none` the `file` element. After testing the accessibility with VoiceOver it appears that the result is the same if I compare it to the previous version.

Just wondering if I am misinterpreted your comment or not?